router/router_test.go

```diff
@@ -0,0 +1,219 @@
+package router
+
+import (
+	"academy/mocks"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_InitServer(t *testing.T) {
+	testCases := []struct {
+		name           string
+		endpoint       string
+		handlerName    string
+		status         int
+		callController bool
+	}{
+		{
+			name:           "OK, Get Jokes",
+			endpoint:       "/api/v1/jokes",
+			handlerName:    "JokesGetJokes",
+			status:         200,
```

Comment:
Make sure you're using the http constants also in test

Reply:
Done